tests/validation/tests/v3_api/test_airgap.py

```diff
@@ -763,6 +767,107 @@ def setup_ssh_key(bastion_node):
+                    '["<registry_host>"], "insecure-registries": ["<registry_host>"]}'
+    bastion_node = get_bastion_node(BASTION_ID)
+    docker_config = docker_config.replace('<registry_host>',
+                                          bastion_node.host_name)
+    node_name = AG_HOST_NAME + "-agwin"
+    aws = AmazonWebServices()
+    win_node = aws.create_node(node_name, ssh_user='Administrator')
+    aws.wait_for_node_state(node=win_node)
+    time.sleep(60 * 6)
+    win_node.ssh_password = aws.decrypt_windows_password(win_node.provider_node_id)
+    win_path = "C:\\ProgramData\\docker\\config\\daemon.json"
+    win_command = "echo {0} > {1}".format(docker_config, win_path)
+    check_windows_command_output(win_node, win_command, error='')
+    win_command = 'powershell -command "restart-service docker"'
+    check_windows_command_output(win_node, win_command, error='')
+    win_sac_version_cmd = 'powershell -command "Get-ItemProperty'\
+                          ' \'HKLM:\\SOFTWARE\\Microsoft\\Windows '\
+                          'NT\\CurrentVersion\\\' | '\
+                          'Select-Object -ExpandProperty ReleaseId"'
+    out = check_windows_command_output(win_node, win_sac_version_cmd, error='')
+    print(out)
+    win_sac_version = out[0]
+    win_sac_version = "".join(filter(str.isalnum, win_sac_version))
+
+    print('==== DOWNLOAD RELEASE SCRIPTS ====')
+
+    win_command = 'powershell -NonInteractive -command '\
+                  '"$ProgressPreference=\'SilentlyContinue\'; '\
+                  'Invoke-WebRequest '\
+                  '\'https://github.com/rancher/rancher/releases/'\
+                  'download/{0}/rancher-windows-images.txt\' '\
+                  '-OutFile \'.\\rancher-windows-images.txt\''\
+                  .format(RANCHER_SERVER_VERSION)
+    check_windows_command_output(win_node, win_command, output='', error='')
+
+    win_command = 'powershell -NonInteractive -command '\
+                  '"$ProgressPreference=\'SilentlyContinue\'; '\
+                  'Invoke-WebRequest '\
+                  '\'https://github.com/rancher/rancher/releases/'\
+                  'download/{0}/rancher-save-images.ps1\' '\
+                  '-OutFile \'.\\rancher-save-images.ps1\''\
+                  .format(RANCHER_SERVER_VERSION)
+    check_windows_command_output(win_node, win_command, output='', error='')
+
+    win_command = 'powershell -NonInteractive -command '\
+                  '"$ProgressPreference=\'SilentlyContinue\'; '\
+                  'Invoke-WebRequest ' \
+                  '\'https://github.com/rancher/rancher/releases/'\
+                  'download/{0}/rancher-load-images.ps1\' '\
+                  '-OutFile \'.\\rancher-load-images.ps1\''\
+                  .format(RANCHER_SERVER_VERSION)
+    check_windows_command_output(win_node, win_command, output='', error='')
+
+    print('==== SAVE IMAGES ====')
+    win_command = 'powershell -NonInteractive -File .\\rancher-save-images.ps1'
```

Comment:
Can we get this (or maybe a separate function and test) to accept a list of images? Similar to how `test_add_images_to_private_registry` does it?